### PR TITLE
guide for Wallabag

### PR DESCRIPTION
**Pull request by ezzra**
This is a guide how to install the Wallabag server side application.

### CODE REVIEW

File: source/guide_wallabag.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  [isabell@stardust ~]$ git clone https://github.com/wallabag/wallabag.git ~/html
+  Cloning into '/home/isabell/wallabag'...
+  remote: Counting objects: 46655, done.
+  remote: Compressing objects: 100% (23/23), done.
+  remote: Total 46655 (delta 10), reused 20 (delta 9), pack-reused 46620
+  Receiving objects: 100% (46655/46655), 58.15 MiB | 20.81 MiB/s, done.
+  Resolving deltas: 100% (26623/26623), done.
+  [isabell@stardust ~]$
+
+Change to that folder and run ``make install``. During the installation process, you will be asked for quite a lot configuration settings. Bu you only need to set up the following informations:
```
Comment (SalocinHB) [inline source/guide_wallabag.rst]: 'quite a lot configuration settings' -> 'a lot of configuration settings'. 'information' should be singular.

Comment (SalocinHB) [reply source/guide_wallabag.rst]: Oh, and 'Bu' -> 'But'.

File: source/guide_wallabag.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+* ``database_name:`` <username>_wallabag - *replace <username> with the MySQL username*
+* ``database_user:`` put in the MySQL username that you looked up in the prerequisites
+* ``database_password:`` that is the MySQL password that you looked up in the prerequisites
+* ``domain_name:`` put in here your domain or subdomain like https://yourdomain
+* ``secret:`` type in any random string here, do not keep the defaul string!
+* ``twofactor_sender:`` choose a mailaddress that fits as sender
+* ``fosuser_registration:`` set this to false, otherwise anyone can register at your wallabag instance
+* ``from_email:`` choose a mailaddress that fits as sender (can be same as before)
```
Comment (SalocinHB) [inline source/guide_wallabag.rst]: Please change this to 'choose an email address to be used as sender (can be the same as above)'

File: source/guide_wallabag.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  database_user (root): isabell
+  database_password (null): MySuperSecretPassword
+  database_path (null): 
+  database_table_prefix (wallabag_):  
+  database_socket (null): 
+  database_charset (utf8mb4):  
+  domain_name ('https://your-wallabag-url-instance.com'): https://isabell.uber.space
+  mailer_transport (smtp): 
+  mailer_host (127.0.0.1): 
+  mailer_user (null):
```
Comment (SalocinHB) [inline source/guide_wallabag.rst]: This won't work …

Comment (ezzra) [reply source/guide_wallabag.rst]: Mh true, it works on U6 and I thought I did successfully testing it on U7 but its not. I fixed it. 

I would have liked to link to a description how to set the email password in the U7 manual, but did not find any. 
Its just mentioned in the German dashboard//Datenblatt that you have to set the ssh password first.

File: source/guide_wallabag.rst (hunk 2 above)
Comment (ezzra) [reply source/guide_wallabag.rst]: fixed

File: source/guide_wallabag.rst (hunk 1 above)
Comment (ezzra) [reply source/guide_wallabag.rst]: fixed

File: source/guide_wallabag.rst [hunk 4]
```diff
@@ -0,0 +1,147 @@
+.. highlight:: console
+
+########
+Wallabag
+########
+
+Wallabag_ is a read later solution like `Firefox Pocket`_ to save and organize articles between devices and make them offline available. This is the server-side application, it will fetch articles and save the content and images on the server when a link is provided. There is also client software for browsers and mobile devices available which can be used to download the fetched articles and add new articles to the server.
```
[the remark this answers is not in the document]
Comment (ezzra) [reply source/guide_wallabag.rst]: fixed

File: source/guide_wallabag.rst [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+
+.. _Wallabag: https://wallabag.org
+.. _PHP: https://manual.uberspace.de/en/lang-php.html
+.. _MySQL: https://manual.uberspace.de/en/database-mysql.html
+.. _supervisord: https://manual.uberspace.de/en/daemons-supervisord.html
+.. _domains: https://manual.uberspace.de/en/web-domains.html
+.. _GitHub: https://github.com/wallabag/wallabag
+.. _feed: https://github.com/wallabag/wallabag/releases.atom
+.. _`Firefox Pocket`: https://support.mozilla.org/en-US/kb/save-web-pages-later-pocket-firefox
```
Comment (SalocinHB) [inline source/guide_wallabag.rst]: remove the backticks

File: source/guide_wallabag.rst [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+.. _`smtp port`: https://manual.uberspace.de/en/mail-access.html#smtp
```
Comment (SalocinHB) [inline source/guide_wallabag.rst]: remove the backticks

**Comment by jeyca** [comment]
smtp port link is still not working, see https://lab.uberspace.de/en/guide_wallabag.html

The backtickets where not the problem, but the enclosing markdown with \*\*, seems that it is not possible to put a link like that within markdown enclosing...

**Comment by ezzra** [comment]
smtp port link is still not working, see https://lab.uberspace.de/en/guide_wallabag.html

The backtickets where not the problem, but the enclosing markdown with \*\*, seems that it is not possible to put a link like that within markdown enclosing...

**Comment by ezzra** [comment]
https://github.com/Uberspace/lab/pull/73

**Pull request merged**